src/app/pages/details/common/get-this-title-files/give-before-pdf/content-warning.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+export default function ContentWarning({
+    link, track, close, onDownload, warning
+}: {
+    link: string;
+    track: string | undefined;
+    close: () => void;
+    onDownload: (event: React.MouseEvent) => void;
+    warning: string;
+}) {
+    const closeAfterDelay = React.useCallback(
+        (event: React.MouseEvent) => {
+            if (onDownload) {
+                onDownload(event);
+            }
+            window.setTimeout(close, 200);
+        },
+        [close, onDownload]
+    );
+
+    return (
+        <div className="give-before-pdf">
+            <div className="text-content">
+                {warning}
+            </div>
+            <a
+                href={link}
+                {...(track ? {'data-track': track} : {})}
+                onClick={closeAfterDelay}
+                className="btn go-to"
+            >
+                Go to your file
```

Comment:
Could we make this work like the give modal does, so it would say "Go to your book" if that was the resource clicked? If not, maybe "Continue"?

Reply:
I made it use "book" for "View online" but left it "file" for PDF. Should it be "book" for PDF, too?

Reply:
To me it makes sense to match the give modal so I think file is fine?